gluten-ut/spark33/src/test/scala/org/apache/gluten/utils/velox/VeloxTestSettings.scala

```diff
@@ -886,6 +886,7 @@ class VeloxTestSettings extends BackendTestSettings {
     .exclude("replace partial and final hash aggregate together with sort aggregate")
     .exclude("do not replace hash aggregate if child does not have sort order")
     .exclude("do not replace hash aggregate if there is no group-by column")
+    .excludeGlutenTest("replace partial and final hash aggregate together with sort aggregate")
```

Comment:
no, it conflicts with Spark's ReplaceHashWithSortAgg, gluten prefers to MergeTwoPhasesHashBaseAggregate

Reply:
When Spark's `ReplaceHashWithSortAgg` is enabled, Gluten still transforms sort agg to its hash agg by default. This test was actually ported from Spark and the expected number of hash/sort agg nodes were modified to adapt to Gluten's planning. Can we keep this test? Maybe, you need modify the test or simply disable `MergeTwoPhasesHashBaseAggregate` here?

Reply:
I have tried this suite before. To retain this test, we must set `mergeTwoPhasesAggregate` to false and eliminate the `COUNT` test, since it is no longer relevant after the current PR's conversion to HashAggTransformer.

Reply:
ok, will keep this test